src/System.Security.Cryptography.X509Certificates/tests/CollectionTests.cs

```diff
@@ -496,7 +496,7 @@ public static void ImportStoreSavedAsCerData()
         }
 
         [Fact]
-        [PlatformSpecific(PlatformID.Windows)]
+        [PlatformSpecific(Xunit.PlatformID.Windows)]
```

Comment:
`PlatformID` has been renamed `TestPlatforms`. See #12284. Sorry for the merge conflicts.